CMakeLists.txt

```diff
@@ -63,10 +57,8 @@ endif()
 
 set(contrib-pattern "src/contrib/*.cpp")
 set(src-pattern "src/*.cpp")
-if (CMAKE_VERSION VERSION_GREATER 3.12)
-  list(INSERT contrib-pattern 0 CONFIGURE_DEPENDS)
-  list(INSERT src-pattern 0 CONFIGURE_DEPENDS)
-endif()
+list(INSERT contrib-pattern 0 CONFIGURE_DEPENDS)
```

Comment:
You can just put the `CONFIGURE_DEPENDS` in the `file(GLOB` below, this was probably only to conditionally include it.

Reply:
You are right, I removed even more lines.